packages/core/src/fhir-to-cda/cda-templates/components/__tests__/vital-signs-examples.ts

```diff
@@ -0,0 +1,109 @@
+import { Observation } from "@medplum/fhirtypes";
+
+export const vitalSignObservationTemplate: Partial<Observation> = {
+  resourceType: "Observation",
+  status: "final",
+  category: [
+    {
+      coding: [
+        {
+          system: "http://terminology.hl7.org/CodeSystem/observation-category",
+          code: "vital-signs",
+          display: "Vital signs",
+        },
+      ],
+    },
+  ],
+  code: {
+    coding: [
+      {
+        system: "http://loinc.org",
+        code: "72514-3",
+        display: "Pain severity - 0-10 verbal numeric rating [Score] - Reported",
+      },
+    ],
+    text: "Pain severity - 0-10 verbal numeric rating [Score] - Reported",
+  },
+  effectiveDateTime: "2013-09-19T18:14:30-07:00",
+  issued: "2013-09-19T18:14:30.324-07:00",
+  valueQuantity: {
+    value: 0,
+    unit: "{score}",
+    system: "http://unitsofmeasure.org",
+    code: "{score}",
+  },
+};
```

Comment:
1. is pain a PQ value like the rest of them?
2. I am confused still why it has {score} instead of an actual unit like the rest of the constants in this file

Reply:
`score` probably because it's not an actual measurable unit. You it's just a score you assign to how you feel on a 0-10 scale. And ya, I think PQ also makes sense here to make it simple